Data/Sys/ApprovedInis.txt

```diff
@@ -0,0 +1,13 @@
+# The purpose of this file is to provide a list of hashes for approved ini files for RetroAchievements use.
+# Files approved in this list may be used; if any hash is incorrect then it has been modified by the
+# player and risks being used for cheating.
+# When editing any ini file in Sys/GameSettings, check if that file is here. If it is, and the changes you
+# have made do not modify gameplay in a cheating fashion, please recalculate the SHA1 of that file and
+# update it in this list. Then, recalculate the SHA1 of *this* file and update it in AchievementManager.h.
+
+GCCE01 START
+# Fix buffer overrun bug (crash at Goblin Wall)
+6C107FEC15C76201233CA2645EB5FAB4FF9751CE
+# Fix GBA connections
+483BDB94615C690045C3759795AF13CE76552286
+GCCE01 END
```

Comment:
Okay, and I just have to ask... Why did you hack together a janky custom format instead of just using *anything* standard? You can avoid any custom parsing logic by just stuffing this into a json, which would also massively improve readability of the comparison logic, as well as making sure it is free of errors.